hoogle-test/ref/type-sigs/test.txt

```diff
@@ -6,11 +6,11 @@
 
 module ReaderT
 newtype ReaderT r m a
-ReaderT :: r -> m a -> ReaderT r m a
+ReaderT :: (r -> m a) -> ReaderT r m a
```

Comment:
Yikes, I can't believe I didn't notice this before—we even checked in this junk!